File: 1-easy/problems/2784-check-if-array-is-good/solution.ts
function isGood(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: bool
  """
  cnt = [0]*(nums).length
  for (x in nums) {
      if (x < (cnt).length) {
          cnt[x] += 1
      } else {
          return false;
  return all(cnt[x] == 1 for x in Array.from({length: 1, (nums}, (_, i) => i).length-1));
}

export { isGood };

File: 1-easy/problems/2784-check-if-array-is-good/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def isGood(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        cnt = [0]*len(nums)
        for x in nums:
            if x < len(cnt):
                cnt[x] += 1
            else:
                return False
        return all(cnt[x] == 1 for x in range(1, len(nums)-1))
